src/js/search.js

```diff
@@ -15,7 +16,22 @@ const search = (e) => {
             }
         })
         .then((data) => {
-            console.log(data)
+            console.log(data);
+            let x = data;
```

Comment:
niepotrzebne przypisanie. uzyj data zamiast x w dalszym kodzie

Reply:
z data tylko mi nie dzialalo, musialam zapisac w zmiennej x,moze dlatego, ze bylo to w funkcji, moge zostawic? Jak nie to zrobie test czy bedzie dzialac bez x :)